### PR TITLE
Sprint updates

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- sprint updates

### CODE REVIEW

File: deployed/doc-it/R19Roadmap.json [hunk 1]
```diff
@@ -454,7 +454,7 @@
         "description": "Add documentation about the usage of a user menu",
         "year": 2021,
         "quarter": "Q1",
-        "status": "in-progress",
+        "status": "finished",
```
Comment (daileytj) [inline deployed/doc-it/R19Roadmap.json:457]: I've never updated this so I'm not entirely sure about the definition of "finished" and "pre-release." Should this be marked "finished" if it hasn't been deployed?

File: deployed/doc-it/R19Roadmap.json [hunk 2]
```diff
@@ -700,7 +700,7 @@
         "description": "React Native project templates for the CLI",
         "year": 2021,
         "quarter": "Q1",
-        "status": "in-progress",
+        "status": "pre-release",
```
Comment (daileytj) [inline deployed/doc-it/R19Roadmap.json:703]: Similar question... should this be marked "pre-release" if we haven't released a beta of the cli yet?

**Review by daileytj — APPROVED** [review]

File: deployed/doc-it/R19Roadmap.json (hunk 2 above)
Comment (emclaug2) [reply deployed/doc-it/R19Roadmap.json:703]: Hmmm. If we wait until Monday to merge this in then we can get away it being prelease.

File: deployed/doc-it/R19Roadmap.json (hunk 1 above)
Comment (emclaug2) [reply deployed/doc-it/R19Roadmap.json:457]: Yeah that's a good question; it's in dev right now, if we merge dev->master then we can mark it finished.

Comment (joebochill) [reply deployed/doc-it/R19Roadmap.json:457]: I'm currently soliciting feedback from studio blue on this before making the PR into master, so at the moment it is pre-release.

File: deployed/doc-it/R19Roadmap.json (hunk 2 above)
Comment (joebochill) [reply deployed/doc-it/R19Roadmap.json:703]: We should publish official versions of the templates and an updated CLI today.

Comment (emclaug2) [reply deployed/doc-it/R19Roadmap.json:703]: Ok I've updated this status to 'finished' pending the publish.

File: deployed/doc-it/R19Roadmap.json (hunk 1 above)
Comment (emclaug2) [reply deployed/doc-it/R19Roadmap.json:457]: This is now pre-release.

**Review by joebochill — APPROVED** [review]

**Pull request merged**